### ISSUE TITLE
Truncate timefrac up to 6 digits

### ISSUE THREAD

**Issue by StrongestNumber9** [teragrep/k8s_01#53]
```
 Example 5 - An Invalid TIMESTAMP

         2003-08-24T05:14:15.000000003-07:00

   This example is nearly the same as Example 4, but it is specifying
   TIME-SECFRAC in nanoseconds.  This results in TIME-SECFRAC being
   longer than the allowed 6 digits, which invalidates it.
```

**Comment by kortemik** [comment]
this is fixed now

**Comment by StrongestNumber9** [comment]
Should be fixed by #56
